server/services/image-optimizer-service.ts

```diff
@@ -32,7 +32,7 @@ async function optimizeImage(file: SourceFile): Promise<StrapiImageFormat[]> {
   } = settingsService.settings;
 
   const sourceFileType = file.ext.replace(".", "") as keyof FormatEnum;
-  if (exclude.includes(sourceFileType) || !include.includes(sourceFileType)) {
+  if (exclude.includes(sourceFileType) || !include.includes(sourceFileType.toLowerCase())) {
```

Comment:
```suggestion
  if (exclude.includes(sourceFileType.toLowerCase()) || !include.includes(sourceFileType.toLowerCase())) {
```

This should also be done for the excluded file types.